Nim.cs

```diff
@@ -23,14 +30,51 @@ public Nim()
         }
         public void StartGame()
         {
-            Console.WriteLine("Hello! Let's play a game of Nim (21)!");
+            Console.WriteLine("Hello! Let's play a game of Nim!");
+            switch (GetGameType()) {
+                case 21:
+                    Nim21();
+                    break;
+                case 100:
+                    Nim100();
+                    break;
+            }
+
+            GameLoop();
+        }
+
+        public void Nim21()
+        {
+            gameType = Variant.Nim21;
             Console.WriteLine("The game begins with one player saying the number 1, then each player takes turns saying 1, 2, or 3, adding to a running total.");
             Console.WriteLine("The player who is forced to say a number resulting in a total of 21 or greater loses!");
             Console.WriteLine("I'll go first. 1, for a total of 1.");
-            GameLoop();
         }
 
-        private static int GetChoice(int total)
+        public void Nim100()
+        {
+            gameType = Variant.Nim100;
+            Console.WriteLine("The game begins with one player saying a number between 1 and 10 (inclusive), then each player takes turns saying a number between 1 and 10 (inclusive), adding to a running total.");
+            Console.WriteLine("The player who says 100 or greater first wins!");
+            var random = new Random();
+            var choice = random.Next(1, 11);
+            Total = choice;
+            Console.WriteLine("I'll go first. {0}, for a total of {1}.", choice, Total);
+
+        }
+
+        private static int GetGameType()
+        {
+            int choice;
+            do
+            {
+                Console.WriteLine("Would you like to play a game of Nim 21 or Nim 100? Enter 21 for Nim 21, and 100 for Nim 100");
+                choice = int.Parse(Console.ReadLine());
```

Comment:
I believe that this will throw if the user enters something that isn't a valid integer and that `TryParse` would be preferable.